addons/website/views/snippets/s_cover.xml

```diff
@@ -2,14 +2,15 @@
 <odoo>
 
 <template id="s_cover" name="Cover">
-    <section class="s_cover parallax s_parallax_is_fixed bg-black-50 pt96 pb96" data-scroll-background-ratio="1">
-        <span class="s_parallax_bg oe_img_bg" style="background-image: url('/web/image/website.s_cover_default_image'); background-position: 50% 0;"/>
+    <section class="s_cover parallax s_parallax_is_fixed o_cc o_cc4 bg-black-50 pt232 pb232" data-scroll-background-ratio="1">
```

Comment:
Since you added a color preset, `bg-black-50`is not needed anymore ;)

```suggestion
    <section class="s_cover parallax s_parallax_is_fixed o_cc o_cc4 pt232 pb232" data-scroll-background-ratio="1">
```